src/mgos_mqtt_conn.c

```diff
@@ -638,7 +638,9 @@ bool mgos_mqtt_conn_unsub(struct mgos_mqtt_conn *c, const char *topic) {
     if (0 == strcmp(s->topic.p, topic)) {
       LOG(LL_INFO,
           ("MQTT%d unsub %.*s", c->conn_id, (int) s->topic.len, s->topic.p));
-      // mg_mqtt_unsubscribe(c->nc, (char **)&topic, 1, mgos_mqtt_conn_get_packet_id(c));
+      if (c->connected)
```

Comment:
please add braces here, we don't do multiline braceless ifs 'round 'ere :)

Reply:
Obliged, gov'na!